tensorflow/core/lib/io/record_reader.cc

```diff
@@ -204,6 +208,24 @@ Status RecordReader::ReadRecord(uint64* offset, tstring* record) {
   return Status::OK();
 }
 
+Status RecordReader::SkipRecords(uint64* offset, int num_to_skip) {
```

Comment:
can this return the number of elements skipped? I could see the called wanting to know that in some cases. That could be more useful than OUT_OF_RANGE